.ruby-version

```diff
@@ -0,0 +1 @@
+2.6.3
```

Comment:
why don't use 2.6.5 ?

Reply:
Ruby 2.6.5 was released *today* I don't know if travis-ci has support for it yet.